### ISSUE TITLE
Example #3 in rrdcreate.pod doesn't work

### ISSUE THREAD

**Issue by tgerdes** [oetiker/rrdtool-1.x#590]
The example in https://github.com/oetiker/rrdtool-1.x/blob/a0e9ed27/doc/rrdcreate.pod#example-3 Does not work:
```
tgerdes@tgerdes-mbpr:~ $ rrdtool create proxy.rrd --step 300 \
>    DS:Total:DERIVE:1800:0:U  \
>    DS:Duration:DERIVE:1800:0:U  \
>    DS:AvgReqDur:COMPUTE:Duration,Requests,0,EQ,1,Requests,IF,/ \
>    RRA:AVERAGE:0.5:1:2016
ERROR: failed to parse computed data source
```

I think the references to "Requests" should actually be "Total";  The description of the example also refers to things by different names than the actual PDPs.

**Comment by oetiker** [comment]
would you like to test the example and supply a patch for the tutorial? that would be great!

**Comment by oetiker** [comment]
fixed 9b4cd0d
